### PR TITLE
Fix import issues and add first placeholder test

### LINKED ISSUE #23: Get first placeholder test working
The current folder name `feature-selection` cannot be imported in Python code and needs to be renamed.

[X] Rename folder
[X] Convert RFE class back into function
[X] Get dummy test working

### PR DESCRIPTION
**Pull request by techrah**

### CODE REVIEW

File: feature_selection/ForwardSelect.py [hunk 1]
```diff
@@ -19,22 +19,22 @@ class ForwardSelect:
     -------
     features_ : numpy ndarray
       Boolean array of selected features
-      
+
     Examples
     --------
-    >>> from feature-selection-python import ForwardSelect
+    >>> from feature_selection-python import ForwardSelect
```
Comment (vcuspinera) [inline feature_selection/ForwardSelect.py]:
```suggestion
    >>> from feature_selection import ForwardSelect
```

File: feature_selection/simulated_annealing.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ class simulated_annealing:
 
 	Examples:
 	---------
-	>>> from feature-selection-python import simulated_annealing
+	>>> from feature_selection-python import simulated_annealing
```
Comment (vcuspinera) [inline feature_selection/simulated_annealing.py]:
```suggestion
	>>> from feature_selection import simulated_annealing
```

**Review by vcuspinera — CHANGES_REQUESTED** [review]

File: feature_selection/ForwardSelect.py (hunk 1 above)
Comment (techrah) [reply feature_selection/ForwardSelect.py]: good catch!

**Review by vcuspinera — APPROVED** [review]
Ok, looks good

**Pull request merged**